go/vt/vtorc/inst/analysis_dao.go

```diff
@@ -475,6 +477,9 @@ func GetReplicationAnalysis(keyspace string, shard string, hints *ReplicationAna
 			a.Analysis = PrimarySemiSyncMustNotBeSet
 			a.Description = "Primary semi-sync must not be set"
 			//
+		} else if topo.IsReplicaType(a.TabletType) && a.ErrantGTID != "" && config.ConvertTabletWithErrantGTIDs() {
```

Comment:
The detection should not be conditioned by the course of action. Analysis should be made for `ErrantGTIDDetected` irrespective of  `config.ConvertTabletWithErrantGTIDs()`.

Reply:
Fixed.